### PR TITLE
bump go-deps

### PR DESCRIPTION
**Pull request by techknowlogick**

### CODE REVIEW

**Comment by delvh** [comment]
Backport 1.21?

**Comment by techknowlogick** [comment]
> Backport 1.21?

Thanks for the suggestion. That's a good idea. I've added the label :)

File: go.mod [hunk 1]
```diff
@@ -303,7 +300,7 @@ replace github.com/hashicorp/go-version => github.com/6543/go-version v1.3.1
 
 replace github.com/shurcooL/vfsgen => github.com/lunny/vfsgen v0.0.0-20220105142115-2c99e1ffdfa0
 
-replace github.com/nektos/act => gitea.com/gitea/act v0.243.4
+replace github.com/nektos/act => gitea.com/gitea/act v0.2.51
```
Comment (delvh) [inline go.mod:303]: Erm… Isn't that a downgrade?

**Review by delvh — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (techknowlogick) [reply go.mod:303]: 2.51 is newer than 243.4 See: https://github.com/nektos/act/releases

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
As already mentioned in the backport, it seems `go-cron` bump introduced this build failure:

https://github.com/go-gitea/gitea/actions/runs/6441499888/job/17491365648?pr=27501

**Pull request merged**